java/flight/flight-core/src/main/java/org/apache/arrow/flight/grpc/StatusUtils.java

```diff
@@ -121,6 +122,17 @@ public static FlightStatusCode fromGrpcStatusCode(Status.Code code) {
     }
   }
 
+  /** Create Metadata Key for binary metadata. */
+  public static Metadata.Key<byte[]> keyOfBinary(String name) {
```

Comment:
Does this need to be public?

Reply:
No, good catch. I was going to update it in other places, but it didn't seem necessary. I'll change to private.